Standards/scs-0003-v1-sovereign-cloud-standards-yaml.md

```diff
@@ -10,34 +10,62 @@ track: Global
 The [Sovereign Cloud Stack (SCS)](https://scs.community) provides standards for a range of cloud infrastructure types.
 It strives for interoperable and sovereign cloud offerings which can be deployed and used by a wide range of organizations and individuals.
 
-With a growing number of documents and certification levels for Sovereign Cloud Stack, it is becoming increasingly difficult to keep track
-of all stable, obsoleted as well as all upcoming standards. This decision record describes how we use technically describe
-certification levels and the mandatory standards therein.
+SCS plans to offer six types of certificates, which are sorted into two dimensions:
+
+1. _certification level_, of which there are three:
+   - SCS-compatible
+   - SCS-open
+   - SCS-sovereign
+2. _cloud layer_, of which there are two:
+   - infastructure as a service (IaaS)
+   - Kubernetes as a service (KaaS)
+
+So, for instance, a certificate can be of type _SCS-compatible IaaS_ or _SCS-sovereign KaaS_.
+
+Each type of certificate amounts to a set of standards that have to be fulfilled by the cloud service in question in order for a certificate to be issued.
+In addition, a certificate of a certain type may only be issued if some other certificate is already held.
+Case in point: the certification levels are meant to be seen as a progression, where the upper levels build on the lower ones, and
+the certificate for "SCS-open IaaS" will only be issued if a certificate for "SCS-compatible IaaS" is already held.
+We say that the latter certificate is a _prerequisite_ of the former.
+
+Naturally, as the state of the art progresses, so do our certificates. We keep track of the changes by means of versioning.
+That is to say that each certificate type can come in several versions, each one of them having its distinct timespan when it is in effect.
+For instance, we might have
+
+- SCS-compatible IaaS v1, effective 2021-01-01 through 2023-10-31
+- SCS-compatible IaaS v2, effective 2023-03-23 through 2023-11-30
+
+and so on (but usually, we aim to keep at most two versions in effect, with an overlap of 4 to 6 weeks).
+
+This decision record describes two main points:
+
+1. How we denote our certificate types by means of a YAML file.
+2. Our process for constructing and progressing the certificate types.
 
 ## Motivation
 
-This decision record has three main objectives:
+This decision record establishes a mechanism (by means of the YAML file) with the following three main objectives:
```

Comment:
Isn't this of type `Procedural` and not of `Decision Record`?

```suggestion
This standard establishes a mechanism (by means of the YAML file) with the following three main objectives:
```

Reply:
The verbiage in question didn't stem from me. Anyway, following your argumentation, we should change "decision record" into "procedural", right? However, I suppose "decision record" could be construed as a kind of general term (a superset) that includes Standards, Procedurals, and any other document. From this point of view, every standard is a decision record, but not every decision record is a standard. If we expressly designate a document as "decision record" in the header, then this mostly means that it doesn't fit into any of the subcategories. Just one way of looking at it.